core/src/main/java/com/linecorp/armeria/internal/logging/ContentPreviewingUtil.java

```diff
@@ -114,7 +114,10 @@ protected ContentPreviewerHttpResponse(
             this.factory = factory;
             this.ctx = ctx;
             whenComplete().handle((unused, cause) -> {
-                if (responseContentPreviewer != null) {
+                if (responseContentPreviewer == null) {
+                    return null;
```

Comment:
I assume this change is to ensure failed responses (e.g. `HttpResponse#ofFailure`) do not record `log#[request|response]ContentPreview`.

Reply:
Right. I thought that response content preview of the failed response is set here:

https://github.com/line/armeria/pull/6269/files/5e540da155391cc16073c77e88dcc08259a0f1b0